model/connect/PDOConnector.php

```
@@ -178,7 +179,7 @@ public function connect($parameters, $selectDB = false) {
 	}
 
 	public function getVersion() {
-		return $this->pdoConnection->getAttribute(PDO::ATTR_SERVER_VERSION);
+		return @$this->pdoConnection->getAttribute(PDO::ATTR_SERVER_VERSION);
```

Why do we need to suppress errors here?